src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -965,7 +967,11 @@ export function isFiltered(
             (!filter.patientTreatmentGroupFilters ||
                 _.isEmpty(filter.patientTreatmentGroupFilters.filters)) &&
             (!filter.sampleTreatmentGroupFilters ||
-                _.isEmpty(filter.sampleTreatmentGroupFilters.filters)))
+                _.isEmpty(filter.sampleTreatmentGroupFilters.filters)) &&
```

Comment:
there's a few places in this PR where code could benefit from the new optional chaining feature of js https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining